src/pages/home/report/ReportActionsList.tsx

```diff
@@ -417,19 +418,16 @@ function ReportActionsList({
         (isFromCurrentUser: boolean) => {
             // If a new comment is added and it's from the current user scroll to the bottom otherwise leave the user positioned where
             // they are now in the list.
-            if (!isFromCurrentUser) {
+            if (!isFromCurrentUser || !isReportScreenTopmostCentralPane()) {
```

Comment:
I think we don't need to create `isReportScreenTopmostCentralPane` function, we can use `isSearchTopmostCentralPane` like this. It is safe enough

```suggestion
            if (!isFromCurrentUser || isSearchTopmostCentralPane()) {
```

Reply:
Or is there any specific problem that requires us to create isReportScreenTopmostCentralPane function?

Reply:
I wanted it to be future-proof. The reason we can use isSearchTopmostCentralPane now is b/c we now only have two (search/report screen) screens in central pane but the proper check we want to apply here is report screen is in central pane that will prevent future similar bugs if we start adding other screens as central pane screens.

Reply:
@FitseTLT Thanks for the explanation. But we can't know If we will have a new screen in the future, should we need to call notify new action for that new screen.

Reply:
Hmm anyway, the current code also looks fine